infrastructure/lambda/private-api.yaml

```diff
@@ -215,33 +215,6 @@ paths:
         contentHandling: "CONVERT_TO_TEXT"
         type: "aws_proxy"
 
-  /addresses/v2:
-    get:
-      parameters:
-        - $ref: "#/components/parameters/SessionIdHeaderWithUnderscore"
-      responses:
-        "400":
-          $ref: "#/components/responses/Error400"
-        "500":
-          $ref: "#/components/responses/Error500"
-        "200":
-          description: "200 response"
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/PostcodeLookupResponse"
```

Comment:
Probably should have been done before, but this response schema isn't actually correct for the /v2 endpoint. Please could we update?